src/test/Login.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Frontend tests are present, but some of them are not passing:

![image](https://github.com/codeWithHyder/house-rent-frontend/assets/57421823/619e643d-7b88-4ba6-afc0-0cdbbf47224b)

![image](https://github.com/codeWithHyder/house-rent-frontend/assets/57421823/42824bb6-d872-44d1-bd9f-730d99ad4753)


Kindly fix them so all of your tests pass. 👍